File: cmd_balance.py
# cmd_balance.py
from db_manager import get_user_gold, get_user_points

COMMANDS = ["فلوسي", "نقاطي"]

def handle(bot, update):
    text = update.message.text.strip()
    if not text or text not in COMMANDS:
        return

    uid = update.message.from_user.id
    gold = get_user_gold(uid) or 0
    points = get_user_points(uid) or 0

    if gold == 0 and points == 0:
        bot.send_message(
            chat_id=update.message.chat.id,
            text="⌔︙ليس لديك نقاط أو فلوس، أرسل الألعاب وابدأ اللعب!"
        )
    else:
        bot.send_message(
            chat_id=update.message.chat.id,
            text=f"⌔︙عدد نقاطك: {points}\n فلوسك: {gold}"
        )

File: cmd_balance.js
// cmd_balance.js
const { get_user_gold } = require('./db_manager');

const COMMANDS = ["فلوسي", "فلوس", "رصيدي", "رصيدي"];

function handle(bot, ctx) {
    const text = ctx.message?.text?.trim();
    if (!text || !COMMANDS.includes(text)) return;

    const uid = ctx.from.id;
    const gold = get_user_gold(uid);

    ctx.reply(`💰 رصيدك الحالي: ${gold} ذهب`);
}

module.exports = { handle };
